### PR TITLE
Fix the go_package option on the .proto files

### PR DESCRIPTION
**Pull request by Frassle**
# Description

This doesn't really effect anything with our current usage, but if we ever put proto files in another package and try to import the current set it wouldn't actually be able to find them.

I noticed this while working on
https://github.com/pulumi/pulumi/pull/10792 where I added a new "pulumirpc.engine" package. The proto file could refer to things like "pulumirpc.PluginDependency" but the go code then tried to import it like `_go "github.com/pulumi/pulumi/v3/proto/go/pulumirpc"` which isn't actually the correct import path (missing the sdk folder part, and there isn't actually a folder called pulumirpc).

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works

- [ ] I have run `make changelog` and committed the `changelog/pending/<file>` documenting my change

- [ ] Yes, there are changes in this PR that warrants bumping the Pulumi Service API version

### CODE REVIEW

File: proto/pulumi/analyzer.proto [hunk 1]
```diff
@@ -20,7 +20,7 @@ import "google/protobuf/struct.proto";
 
 package pulumirpc;
 
-option go_package = "github.com/pulumi/pulumi/v3/proto/go/pulumirpc";
+option go_package = "github.com/pulumi/pulumi/sdk/v3/proto/go;pulumirpc";
```
Comment (iwahbe) [inline proto/pulumi/analyzer.proto:23]: I'm assuming that the `;` means import as.

**Review by iwahbe — APPROVED** [review]
LGTM

File: proto/pulumi/analyzer.proto (hunk 1 above)
Comment (Frassle) [reply proto/pulumi/analyzer.proto:23]: The compiler uses it to set the package option at the top of the generated files. If your missing it you get "package _go".

**Comment by Frassle** [comment]
bors merge

**Pull request merged**